src/pyimpspec/data/formats/par.py

```diff
@@ -0,0 +1,91 @@
+# pyimpspec is licensed under the GPLv3 or later (https://www.gnu.org/licenses/gpl-3.0.html).
+# Copyright 2023 pyimpspec developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+# The licenses of pyimpspec's dependencies and/or sources of portions of code are included in
+# the LICENSES folder.
+
+from os.path import exists
+from typing import (
+    IO,
+    List,
+)
+from pyimpspec.data.data_set import (
+    DataSet,
+    dataframe_to_data_sets,
+)
+
+
+def parse_par(path: str) -> List[DataSet]:
+    """
+    Parse a Parstat .par file containing an impedance spectrum.
```

Comment:
Is the .par file format specific to VersaStudio?

Reply:
As far as I know, yes.